external/VERSIONS.md

```diff
@@ -1,5 +1,5 @@
-- cxxopts: v3.2.0
-- dmon: 1.3.7
-- imgui: v1.91.9b
+- cxxopts: v3.3.1
+- dmon: 1.3.8
+- imgui: v1.92.2
```

Comment:
`nlohmann_json` and `tinyfiledialogs` are up-to-date?

Reply:
yes, of course